src/client/pages/OfferNew/Introduction/HeroOfferDetails.tsx

```diff
@@ -5,7 +5,9 @@ import { Address } from 'data/graphql'
 import { OfferData } from 'pages/OfferNew/types'
 import { useTextKeys } from 'utils/textKeys'
 import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { TextButton } from 'components/buttons'
 import { getHouseholdSize } from '../utils'
+import { useDetailsModalState } from '..'
```

Comment:
A bit unnecessary for a context just to drill down one step, but it can be used elsewhere